internal/testing/customize/evaluator/evaluator.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
```

Comment:
why are these not just a move from the other location instead of a new file with same content?